airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -73,12 +74,27 @@ protected JdbcDatabase getDatabase(JsonNode config) {
 
   @Override
   public JsonNode toJdbcConfig(JsonNode config) {
+    final List<String> additionalParameters = new ArrayList<>();
+
+    if (config.has("ssl") && config.get("ssl").asBoolean()) {
```

Comment:
can you make SSL the default if it is not specified? This will make it easier to create a secure fork of the connector in #6423

Reply:
@sherifnada, I changed default ssl value to true in spec